istioctl/cmd/proxystatus.go

```diff
@@ -244,6 +243,8 @@ Retrieves last sent and last acknowledged xDS sync from Istiod to each Envoy in
 	centralOpts.AttachControlPlaneFlags(statusCmd)
 	statusCmd.PersistentFlags().StringVarP(&configDumpFile, "file", "f", "",
 		"Envoy config dump JSON file")
+	statusCmd.PersistentFlags().BoolVar(&centralOpts.XdsViaAgents, "xds-via-agents", false,
```

Comment:
it's safe to set default value to `true` on an experimental cmd, which will give us more feed back?

Reply:
This is an option for the special case: istiods are not in the local k8s cluster, so cannot be reached directly. If not in such case, the original approach is efficient.

If this PR is approved and submitted, after some period of tests, I would like to make a subsequent PR to apply this way when there is no "istiod" in the k8s cluster, which means "external istiod".

Reply:
make sense